ember-cli-build.js

```diff
@@ -18,12 +19,10 @@ module.exports = function (defaults) {
 
   const { maybeEmbroider } = require('@embroider/test-setup');
   return maybeEmbroider(app, {
-    packagerOptions: {
-      webpackConfig: {
-        node: {
-          crypto: 'empty',
```

Comment:
Is this not needed anymore?

Reply:
@scalvert actually this was giving me hard error:

```
Invalid configuration object. Webpack has been initialized using a configuration object that does not match the API schema.
51
 - configuration[0].node has an unknown property 'crypto'. These properties are valid:
52
   object { __dirname?, __filename?, global? }
53
   -> Options object for node compatibility features.
```

Looking into [PR where it was introduced](https://github.com/ember-a11y/ember-a11y-testing/pull/264) seem like it was needed to fix breaking build. but now it causes failure 🙃

Reply:
Ya, we can remove it and see if it breaks anything :)